x-pack/plugins/transform/public/app/sections/create_transform/components/step_define/common/percentiles_agg/config.ts

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { PercentilesAggForm } from './percentiles_form_component';
+import { IPivotAggsConfigPercentiles } from './types';
+import {
+  isPivotAggsConfigWithUiSupport,
+  PERCENTILES_AGG_DEFAULT_PERCENTS,
+  PivotAggsConfigBase,
+} from '../../../../../../common';
+import { PivotAggsConfigWithUiBase } from '../../../../../../common/pivot_aggs';
+
+/**
+ * TODO this callback has been moved.
```

Comment:
Is this TODO needed?

Reply:
yes, because the validation logic isn't part of this PR, but it needs some improvements.

Reply:
created an issue https://github.com/elastic/kibana/issues/138874